product_variant_configurator/README.rst

```diff
@@ -0,0 +1,85 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Product Variant Configurator
+============================
+
+Provides an abstract model for product variant configuration.
+
+Configuration
+=============
+
+To configure the creation of the variants behaviour, you need to:
+
+#. Go to ``Warehouse > Products``, and select a product.
```

Comment:
This menu is not available by default, but there's no menu with the base product. But it's better to say that installing `sale` module, you have to:
```
#. Go to Sales > Settings > Configuration, and select "Products can have several attributes, defining variants (Example: size, color,...)" on "Product Variants" section.
#. Go to ``Sales > Products``, and select...
```